posts/sgx_StoryTell.md

```diff
@@ -49,6 +49,48 @@ The Fortanix SGX ABI (compiler target x86_64-fortanix-unknown-sgx) is an interfa
 
 It applies the microcode updates supplied by Intel® and disables hyperthreading on all systems. This prevents unauthorized access to the memory of SGX enclaves through side-channel attacks such as the Foreshadow vulnerability.
 
```

Comment:
I think this section needs its own title, something like "### How to run Gramine"

Reply:
I think that a section on Gramine's execution we should do it on all scenarios and it could be part of the second part of the BlogPost.